ros-rviz-image.html

```diff
@@ -0,0 +1,173 @@
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../paper-checkbox/paper-checkbox.html">
+<link rel="import" href="../paper-dropdown-menu/paper-dropdown-menu.html">
+<link rel="import" href="../paper-input/paper-input.html">
+<link rel="import" href="../paper-listbox/paper-listbox.html">
+<link rel="import" href="imports.html">
+
+<dom-module id="ros-rviz-image">
+  <template>
+    <paper-input label="Image topic" value="{{topic}}"></paper-input>
+    <paper-dropdown-menu id="transport_hints_dropdown" label="Transport hints" on-iron-select="transportHintSelected">
+      <paper-listbox slot="dropdown-content" class="dropdown-content" selected="0">
+        <paper-item>Compressed</paper-item>
+        <paper-item>CompressedDepth</paper-item>
+      </paper-listbox>
+    </paper-dropdown-menu>
+    <div style="position: relative;">
+      <div id="canvas_div" style="position: fixed; z-index: 9; border: 1px solid; background-color: black;">
```

Comment:
How about removing the black bg and border? This way, if an image has a different aspect ratio, the extra black doesn't show. Something like this:

```suggestion
      <div id="canvas_div" style="position: fixed; z-index: 9; background-color: transparent;" title="[[topic]]">
```

Notice I also threw in the title in there, so it's easier to identify which image is which:

![transparent_img](https://user-images.githubusercontent.com/5751272/48876681-3bebb780-edb4-11e8-942b-5d880f0b4787.gif)